### ISSUE TITLE
A False Negative about UCF_USELESS_CONTROL_FLOW

### ISSUE THREAD

**Issue by Zustin** [spotbugs/spotbugs#1847]
Hi, I found a false positive about the rule [UCF_USELESS_CONTROL_FLOW](https://spotbugs.readthedocs.io/en/latest/bugDescriptions.html#ucf-useless-control-flow-ucf-useless-control-flow), please refer to the following code sample. I think this FN is caused by **nested if statemnet**. SpotBugs should recognize tha the inner if statement has empty branch and report a warning at line , but it doesn't.

```java
public static void main(String[] argv) {
    if (argv.length == 1)
      if (true)
          ;  // should report a warning here because this is a empty branch
    System.out.println("Hello, " + argv[0]);
}
```
SpotBugs Version: 4.5.0
Expected Result: Report a warning at line 4, but it doesn't.

The following case can be detected:
```java
public static void main(String[] argv) {
    if (argv.length == 1)
          ;  // SpotBugs reports a warning here
    System.out.println("Hello, " + argv[0]);
}
```
